src/profile/notification-settings/NotificationSettings.ts

```diff
@@ -0,0 +1,184 @@
+
+// "Headers" for each section (note that checkboxes have aria labels to make up for the fact that the information
+// is not actually being displayed in a table).
+const headerGridStyle = { display: 'grid', gridTemplateColumns: 'repeat(4, 1fr)', margin: '1.5rem 0 0.5rem 0' };
+const boldDiv = (text) => div({ style: { fontWeight: 600 } }, text);
+
+const NotificationCardHeaders = memoWithName('NotificationCardHeaders', () => {
+  return div({ style: headerGridStyle }, [
+    div({ style: { gridColumnStart: '1', gridColumnEnd: '3', marginLeft: '0.75rem' } }, [boldDiv('Name')]),
+    div({ style: { textAlign: 'center' } }, [boldDiv('Opt In')]),
+  ]);
+});
+
+const WorkspaceNotificationCardHeaders = memoWithName('NotificationCardHeaders', () => {
+  return div({ style: headerGridStyle }, [
+    div({ style: { gridColumnStart: '1', gridColumnEnd: '3', marginLeft: '0.75rem' } }, [boldDiv('Name')]),
+    div({ style: { display: 'flex', justifyContent: 'center' } }, [
+      boldDiv('Submission Opt In'),
+      h(InfoBox, { style: { marginLeft: '0.5rem' } }, [workspaceSubmissionNotificationInfo]),
+    ]),
+    div({ style: { display: 'flex', justifyContent: 'center' } }, [
+      boldDiv('Changed Opt In'),
+      h(InfoBox, { style: { marginLeft: '0.5rem' } }, [workspaceChangedNotificationInfo]),
+    ]),
+  ]);
+});
+
+export const NotificationSettings = () => {
+  const { workspaces } = useWorkspaces();
+  const [prefsData] = _.over(_.pickBy)((_v, k) => _.startsWith('notifications/', k), userStore.get().profile);
+  const [saving, setSaving] = useState(false);
+
+  const userAttributes = userStore.get().terraUserAttributes;
+  const [marketingConsent, setMarketingConsent] = useState(userAttributes.marketingConsent);
+
+  useEffect(() => {
+    setMarketingConsent(userAttributes.marketingConsent);
+  }, [userAttributes.marketingConsent]);
+
+  return h(PageBox, { role: 'main', style: { flexGrow: 1 }, variant: PageBoxVariants.light }, [
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, margin: 0, textTransform: 'uppercase' } }, [
+        'System Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+          'You may opt in or out of recieving marketing communications from Terra',
+        ]),
+      ]),
+    ]),
+    h(NotificationCardHeaders),
+    div({ role: 'list', 'aria-label': 'communication preferences from terra', style: { flexGrow: 1, width: '100%' } }, [
+      h(UserAttributesCard, {
+        value: true,
+        label: 'Necessary communications related to platform operations',
+        setSaving,
+        notificationKeys: [],
+        notificationType: 'PlatformOperations',
+        disabled: true,
+      } as UserAttributesCardProps),
+      h(UserAttributesCard, {
+        value: marketingConsent,
+        label: 'Marketing communications (including upcoming workshops and new flagship dataset additions)',
```

Comment:
I shortened this because previously it easily got cut off if your browser wasn't super wide. The previous text was: "Marketing communications including notifications for upcoming workshops and new flagship dataset additions"

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/e42a8bf6-f27e-4947-9475-b8fa58ffd0d4)

Reply:
Sure, I can remove the parens